web-api/terraform/template/streams.tf

```diff
@@ -25,4 +25,10 @@ resource "aws_lambda_event_source_mapping" "streams_mapping" {
   event_source_arn  = aws_dynamodb_table.efcms-east.stream_arn
   function_name     = aws_lambda_function.zip_streams.arn
   starting_position = "TRIM_HORIZON"
+
+  # The below config was added due to a potential Terraform bug
+  # see https://github.com/terraform-providers/terraform-provider-aws/issues/14522
+  maximum_batching_window_in_seconds = 0
+  maximum_record_age_in_seconds      = 604800
+  maximum_retry_attempts             = 10000
```

Comment:
Hmm… is this an accidental add? Or, what was this added to fix? The [commit that added it](https://github.com/ustaxcourt/ef-cms/pull/387/commits/bb22c05a8247c5f97af693b7028e419744356f68) seems to also introduce another change which has been reverted.

Reply:
I will remove this with the caveat that it's possible we run into the issue again when creating a fresh environment. Not sure about that, though.

Reply:
removed https://github.com/ustaxcourt/ef-cms/pull/387/commits/2ccee77432d3bb501dececba6a95ab70e9b68406#diff-cc38121f0a101dec01859ae2f231227f